.github/workflows/release-desktop-win.yml

```
@@ -95,24 +94,42 @@ jobs:
+            ./apps/desktop/build-electron/*-arm64.exe
+
+      - name: Upload universal Windows exe
         uses: actions/upload-artifact@v4
         with:
-          name: onekey-desktop-windows-${{ matrix.arch }}
+          name: onekey-desktop-win-universal-exe
           path: |
             ./apps/desktop/build-electron/*.exe
+            !./apps/desktop/build-electron/*-x64.exe
+            !./apps/desktop/build-electron/*-arm64.exe
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)